### PR TITLE
SUMO232354: Update doc for 10 v2 apps 

### PR DESCRIPTION
**Pull request by priyansh-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ Yes] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/cato-networks.md [hunk 1]
```diff
@@ -94,20 +94,13 @@ _sourceName="source" admin
 
 Follow the instructions for setting up [Cato Networks](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source/) source and use the same source category while installing the app.
 
-## Installing the Cato Networks app​
+## Installing the Cato Networks App
```
Comment (JV0812) [inline docs/integrations/saas-cloud/cato-networks.md:97]:
```suggestion
## Installing the Cato Networks app
```

File: docs/integrations/saas-cloud/cisco-umbrella.md [hunk 2]
```diff
@@ -64,20 +64,13 @@ By having all your logs uploaded to an S3 bucket, you can then download logs aut
 
 For more information, see [Cisco Umbrella Documentation](https://docs.umbrella.com/managed-services/docs/msc-manage-logs).
 
-## Installing Cisco Umbrella App
+## Installing the Cisco Umbrella App
```
Comment (JV0812) [inline docs/integrations/saas-cloud/cisco-umbrella.md:67]:
```suggestion
## Installing the Cisco Umbrella app
```

File: docs/integrations/saas-cloud/citrix-cloud.md [hunk 3]
```diff
@@ -68,17 +68,11 @@ sourceCategory="citrixCloudSource"
 
 ## Installing the Citrix Cloud App
```
Comment (JV0812) [inline docs/integrations/saas-cloud/citrix-cloud.md:69]:
```suggestion
## Installing the Citrix Cloud app
```

File: docs/integrations/saas-cloud/cloudflare.md [hunk 4]
```diff
@@ -90,23 +90,13 @@ To enable the Cloudflare Logpush service:
 
 Once connected, Cloudflare lists Sumo Logic as a connected service under Logs > Logpush. Edit or remove connected services from here.
 
-
 ## Installing the Cloudflare App
```
Comment (JV0812) [inline docs/integrations/saas-cloud/cloudflare.md:93]:
```suggestion
## Installing the Cloudflare app
```

File: docs/integrations/saas-cloud/cloudflare.md [hunk 5]
```diff
@@ -90,23 +90,13 @@ To enable the Cloudflare Logpush service:
...
-
-## Viewing the Cloudflare Dashboards
-
-**All dashboard have a set of filters** that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
-You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+{@import ../../reuse/apps/app-install-v2.md}
 
-**Each panel has a set of filters** that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+## Viewing Cloudflare Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/cloudflare.md:97]:
```suggestion
## Viewing Cloudflare dashboards 
```

File: docs/integrations/security-threat-detection/barracuda-waf.md [hunk 6]
```diff
@@ -163,7 +161,7 @@ Now that you have configured log collection for Barracuda WAF, install the Sumo
 
 {@import ../../reuse/apps/app-install.md}
 
-## Viewing Barracuda WAF Dashboards   
+## Viewing Barracuda WAF Dashboards
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/barracuda-waf.md:164]:
```suggestion
## Viewing Barracuda WAF dashboards 
```

File: docs/integrations/security-threat-detection/barracuda-waf.md [hunk 7]
```diff
@@ -153,8 +153,6 @@ parse regex "(?<Unit_Name>[^ ]+) NF(?<Log>.*)"
 | parse field=Log " * * * * * * * * *" as Log_Level, Protocol, Source_Ip, Source_Port, Destination_Ip, Destination_Port, ACL_Policy, ACL_Name, Log_Details
 ```
 
-
-
 ## Install the Barracuda WAF App
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/barracuda-waf.md:156]:
```suggestion
## Install the Barracuda WAF app
```

File: docs/integrations/security-threat-detection/duo-security.md [hunk 8]
```diff
@@ -25,16 +25,13 @@ When you generate the Duo credentials, you should do it for the Admin API applic
 
 To collect logs from the Duo Security App, use the new [Cloud to Cloud Integration for Duo Security App](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/duo-source) to create the source and use the same source category while installing the app.
 
+## Installing the Duo Security App
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/duo-security.md:28]:
```suggestion
## Installing the Duo Security app
```

File: docs/integrations/security-threat-detection/duo-security.md [hunk 9]
```diff
@@ -25,16 +25,13 @@ When you generate the Duo credentials, you should do it for the Admin API applic
...
 
-## Installing the Duo Security App and View the Dashboards
+{@import ../../reuse/apps/app-install-v2.md}
 
-This section provides instructions on how to install the Duo Security App, as well as an example of each of the App dashboards. The App dashboards provide insight into failed and successful authentications, administrator events, and outlier events.
+## Viewing Duo Security Dashboards
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/duo-security.md:32]:
```suggestion
## Viewing Duo Security dashboards
```

File: docs/integrations/saas-cloud/cisco-umbrella.md (hunk 2 above)
Comment (priyansh-sumo) [reply docs/integrations/saas-cloud/cisco-umbrella.md]: Updated

File: docs/integrations/saas-cloud/box.md [hunk 10]
```diff
@@ -406,15 +406,12 @@ The following table shows the **config** parameters for a Box Source.
...
 
-Now that you have set up collection for Box, install the Sumo Logic App for Box to use the preconfigured searches and [dashboards](#viewing-box-dashboards) to analyze your data.
-
-{@import ../../reuse/apps/app-install.md}
-
-The Script Source is available for Linux or Windows environments with Java Runtime Environments.
-
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Box Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/box.md:411]:
```suggestion
## Viewing Box dashboards
```

File: docs/integrations/saas-cloud/cato-networks.md [hunk 11]
```diff
@@ -94,20 +94,13 @@ _sourceName="source" admin
...
-{@import ../../reuse/apps/app-install.md}
-
-## Viewing Cato Networks dashboards​​
-
-* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
- You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+## Viewing Cato Networks Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/cato-networks.md:101]:
```suggestion
## Viewing Cato Networks dashboards 
```

File: docs/integrations/saas-cloud/citrix-cloud.md [hunk 12]
```diff
@@ -66,19 +66,13 @@ sourceCategory="citrixCloudSource"
...
 ```
 
-## Installing the Citrix Cloud App
+## Installing the Citrix Cloud app
 
-Before you begin, collect logs from Citrix Cloud API and ingest them into Sumo Logic. Refer to the [Citrix Cloud API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source/) to create the source and use the same source category while installing the app.
+{@import ../../reuse/apps/app-install-v2.md}
 
-{@import ../../reuse/apps/app-install.md}
+## Viewing Citrix Cloud Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/citrix-cloud.md:73]:
```suggestion
## Viewing Citrix Cloud dashboards
```

**Review by JV0812 — APPROVED** [review]
LGTM

File: README.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 <img src="https://help.sumologic.com/img/sumo-docs-readme.png" width="400"/>
 
+
```
Comment (kimsauce) [inline README.md:3]:
```suggestion
```

File: docs/integrations/security-threat-detection/cylance.md [hunk 14]
```diff
@@ -216,6 +216,7 @@ You can use filters to drill down and examine the data on a granular level.
 **Each panel has a set of filters** that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
 
 
+
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/cylance.md:219]:
```suggestion
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Keep this open; related to apps v2 update

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR #3432.

**Pull request closed without merging**